snappybouncer/tasks.py

```diff
@@ -24,6 +26,50 @@ def send_helpdesk_response(ticket):
     return response
 
 
+def jembi_format_date(date):
+    return date.strftime("%Y%m%d%H%M%S")
+
+
+def extract_class_from_tags(tags):
+    # ["@person", "#coffee", "#payment"] -> "coffee"
+    for tag in tags:
+        if tag[0] == "#":
+            return tag[1::]
+
```

Comment:
I think we should return something here if the ticket wasn't tagged right - maybe default to "other"?